lib/ansible/modules/network/ios/ios_logging.py

```diff
@@ -274,6 +274,8 @@ def map_config_to_obj(module):
     data = get_config(module, flags=['| include logging'])
 
     for line in data.split('\n'):
+        if line.startswith("no "):  # 'no logging console' or similar
```

Comment:
This can be replaced with modifying the regex in L279 to '^logging (\S+)', i.e., fetch lines that strictly start with 'logging'.

Reply:
Done.